### PR TITLE
Jack molosh branch

### PR DESCRIPTION
**Pull request by Moleved**
Task: https://trello.com/c/7FKv6Hsx/23-add-co-users-account-rules-editor-function

### CODE REVIEW

**Comment by kirylpl** [comment]
@kislak Please check

File: app/assets/stylesheets/rules.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Rules controller here.
```
Comment (kislak) [inline app/assets/stylesheets/rules.scss]: Please remove files which are not used

File: app/helpers/rules_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module RulesHelper
```
Comment (kislak) [inline app/helpers/rules_helper.rb]: Remove unused files

File: app/views/invites/index.html.haml [hunk 3]
```diff
@@ -11,4 +11,4 @@
       =User.find(invite.user_to_id).name
       Invited at:
       =invite.created_at
-      -#=link_to 'Delete invite', invite, method: :delete, data: { confirm: 'Are you sure?' }
\ No newline at end of file
+      =link_to 'Delete invite', account_invite_path(invite.account_id, invite), method: :delete, data: { confirm: 'Are you sure?' }
```
Comment (kislak) [inline app/views/invites/index.html.haml]: Add empty line

File: spec/controllers/rules_controller_spec.rb [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+    end
+
+    context 'with invalid params' do
+      it 'does not updates spending limit to negative' do
+        patch :update, params: { account_id: account.id, id: rule.id, account_rule: { spending_limit: -1 } }
+        expect(Rule.find(rule.id).spending_limit).not_to eq(-1)
+      end
+    end
+  end
+end
```
Comment (kislak) [inline spec/controllers/rules_controller_spec.rb]: add empty line

File: spec/factories/rules.rb [hunk 5]
```diff
@@ -0,0 +1,4 @@
+FactoryGirl.define do
+  factory :rule do
+  end
+end
```
Comment (kislak) [inline spec/factories/rules.rb]: add empty line

**Pull request merged**